src/exotic-sizes.md

```diff
@@ -97,7 +125,7 @@ enum Void {} // No variants = EMPTY
 ```
 
 Empty types are even more marginal than ZSTs. The primary motivating example for
-Void types is type-level unreachability. For instance, suppose an API needs to
+an empty type is type-level unreachability. For instance, suppose an API needs to
 return a Result in general, but a specific case actually is infallible. It's
 actually possible to communicate this at the type level by returning a
 `Result<T, Void>`. Consumers of the API can confidently unwrap such a Result
```

Comment:
Should probably change this to `Result<T, !>`, as the standard empty type.

Reply:
! is still unstable and trapped in bikeshed hell, so I don't really have a strong motivation to mention it